lib/events.js

```diff
@@ -0,0 +1,196 @@
+export const petEvents = [
+  {
+    id: 1,
+    eventName: "Your pet is sick",
+    description: `Your pet is sick. You need to pay 100 medical costs and your pet lost 20 energy.`,
+    eventValues: {
+      happiness: 0,
+      hunger: 0,
+      energy: -20,
+      money: -100,
+    },
+  },
+
+  {
+    id: 2,
+    eventName: "Birthday Party",
+    description: `It's your pet birthday and you're throwing a big party with all of his pet friends. Pay 50 pet coins.`,
+    eventValues: {
+      happiness: 100,
+      hunger: 100,
+      energy: 30,
+      money: -50,
+    },
+  },
+  {
+    id: 3,
+    eventName: "Big walk",
+    description: `After a long walk your pet is very happy. However, all the playing and exploring also made your pet tired and hungry. Your happiness increases by 80 but you lose 20 energy and 10 hunger.`,
+    eventValues: {
+      happiness: 80,
+      hunger: 0,
+      energy: -20,
+      money: 0,
+    },
+  },
+  {
+    id: 4,
+    eventName: "Sleepy pet",
+    description: `After a good sleep your pet is full of energy again but also really hungry. You win 60 energy and lose 20 hunger.`,
+    eventValues: {
+      happiness: 0,
+      hunger: -20,
+      energy: 100,
+      money: 0,
+    },
+  },
+  {
+    id: 5,
+    eventName: "Sad news",
+    description: `The doorbell rings and you find yourself face to face with a police officer. Unfortunately, she has to tell you that your pet was hit by a car. Your pet dies.`,
+    eventValues: {
+      happiness: -100,
+      hunger: -100,
+      energy: -100,
+      money: 0,
+    },
+  },
+  {
+    id: 6,
+    eventName: "Archaeological find",
+    description: `your pet digs up an archaeological find. You sell it for 1200 pet coins.`,
+    eventValues: {
+      happiness: 0,
+      hunger: 0,
+      energy: 0,
+      money: 1200,
+    },
+  },
+  {
+    id: 7,
+    eventName: "Mr/s barking",
+    description: `your pet stays up all night barking at the neighbor's cat. Energy decreases by 30.`,
+    eventValues: {
+      happiness: 0,
+      hunger: 0,
+      energy: -30,
+      money: 0,
+    },
+  },
+
+  {
+    id: 8,
+    eventName: "baking hour",
```

Comment:
If all your events start with a capital letter, this should, too.
```suggestion
    eventName: "Baking hour",
```